### PR TITLE
Update README.rdoc

### PR DESCRIPTION
**Pull request by greenbigfrog**

### CODE REVIEW

**Comment by greenbigfrog** [comment]
...

**Comment by jaysonvirissimo** [comment]
Unless I'm reading this wrong, `0.0.7` is the latest release, but this commit references `0.0.8`. Maybe just change it to `0.0.7`?

File: README.rdoc [hunk 1]
```diff
@@ -36,7 +36,7 @@ We assume you already have a ruby 1.9 or 2.0 compatible interpreter and rubygems
 
 if you want to have it available system-wide, just build the gem and install it:
 
- gem build bitcoin-ruby.gemspec && gem install bitcoin-ruby-0.0.5.gem
+ gem build bitcoin-ruby.gemspec && gem install bitcoin-ruby-0.0.8.gem
```
Comment (jaysonvirissimo) [inline README.rdoc]: This seems to fail, but `gem build bitcoin-ruby.gemspec && gem install bitcoin-ruby-0.0.10.gem` succeeds.

**Comment by lian** [comment]
better late than never :) cherry-picked this PR on-top of master. closing this ticket.

**Pull request closed without merging**